Kotlin/app/src/main/java/com/taboola/kotlin/examples/PlacementInfo.kt

```diff
@@ -38,12 +38,31 @@ class PlacementInfo {
         val mode = "thumbs-feed-01"
     }
 
+    class MetaWidgetProperties  {
+        val placementName = "Below Article Thumbnails"
+        val sourceType = "text"
+        val pageType = "article"
+        val pageUrl = "https://blog.taboola.com"
+        val targetType = "mix"
+        val mode = "meta-widget-1x1"
+    }
+
+    class MetaFeedProperties  {
```

Comment:
@voltfmanSlava - Any reason why there is quite a bit of duplication here (sourceType, pageType, pageUrl, targetType)? Can't we have a base/default property set and only have specific mode/placementName?

Reply:
@tomer-br-taboola Done